spec/models/course_spec.rb

```diff
@@ -421,4 +405,67 @@
       end
     end
   end
+
+  describe "#sites_not_associated_with_course" do
+    let(:first_site) { create(:site) }
+    let(:second_site) { create(:site) }
+    let(:provider) { create(:provider, sites: [first_site, second_site]) }
+    let(:first_site_status) { create(:site_status, :findable, site: first_site) }
+    subject { create(:course, provider: provider, site_statuses: [first_site_status]) }
+
+    its(:sites_not_associated_with_course) { should eq([second_site]) }
+  end
+
+  context "adding and removing sites on a course" do
+    let(:provider) { create(:provider) }
+    let(:new_site) { create(:site, provider: provider) }
+    let(:existing_site) { create(:site, provider: provider) }
+    let(:new_site_status) { subject.site_statuses.find_by!(site: new_site) }
+    subject { create(:course, site_statuses: [existing_site_status]) }
+
+    describe "for running courses" do
```

Comment:
This seems to be the `context` here, describing the setup within which tests are running.

Reply:
fixed